src/guillotine/fetchContentData.ts

```diff
@@ -20,7 +16,10 @@ export const fetchContentData = async <Content = Record<string, unknown>>(
     if (variables && Object.keys(variables).length > 0) {
         body.variables = variables;
     }
-    const contentResults = await fetchGuillotine<ContentResult<Content>>(contentApiUrl, body, projectConfig, headers);
+    const contentResults = await fetchGuillotine<ContentResult<Content>>(contentApiUrl, projectConfig, {
```

Comment:
Why not go all the way and make a single "object" parameter?

Reply:
Just changed as little as possible, this was not really necessary here